modules/shortcodes/css/blocks/vr-block.css

```diff
@@ -6,11 +6,15 @@
 	overflow: hidden;
 }
 
+.wp-block-jetpack-vr .components-placeholder__fieldset {
+	justify-content: space-around;
```

Comment:
Seems like this isn't enough - on mobile devices (>320 && <782), sizing and alignment of the fields looks a little odd:

![](https://cldup.com/Dx5FACGxpC.png)